external/upstream/gau2grid/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ else()
     ExternalProject_Add(gau2grid_external
         DEPENDS pybind11_external
         GIT_REPOSITORY https://github.com/dgasmith/gau2grid
-        GIT_TAG v1.1.0
+        GIT_TAG v1.2.0
```

Comment:
May want to bump to 1.2 in line 1 of this file and also `find_package(gau2grid 1.2` cmd in psi4/CMakeLists.txt . It's not strictly req'd for Linux/Mac users, but I think they can `conda update psi4-dev` so everyone can be on the same footing.

Reply:
OK! I have fixed the `gau2grid` version in `find_package`.